routes/index.js

```diff
@@ -6,4 +6,8 @@ router.get('/', (request, response, next) => {
   response.render('index', { title: 'Lizardboard' });
 });
 
+router.get('/styleguide', ( request, response, next ) => {
```

Comment:
Trivial whitespace changes for this route: please insert spaces between parens and params, here and in `response.render`